dist/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/4f02d055e5.js" crossorigin="anonymous"></script>
+    <title>Leaderboard</title>
+<script defer src="/main.js"></script></head>
+<body>
+
+    <h1>Leaderboard</h1>
+    <main>
+        <div class="score-dash">
+            <h2>Recent scores</h2>
+            <input type="button" value="Refresh" id="refresh">
+            <div class="list">
+                <ul>
+                    
```

Comment:
- [ ] index.html main should contain only root element. Kindly try to construct the DOM elements from js files. :pray:

Reply:
Resolved